src/commands/publish.rs

```diff
@@ -112,6 +88,7 @@ pub fn publish(
 
         let upload_client = http::featured_legacy_auth_client(user, Feature::Sites);
 
+        deploy::pre_upload()?;
```

Comment:
Why are we putting this empty function here and elsewhere?

Reply:
`pre_upload()` is for doing tasks that require making an API call before uploading the script. This is going to be used to fetch the current migration tag for tagged migrations, so wrangler can look at the list of migrations in `wrangler.toml` and know how to coalesce them.

I had already done some of the work for tagged migrations and this is part of it, so I figured i'd leave it in